common/lib/xmodule/xmodule/partitions/partitions.py

```diff
@@ -104,6 +104,9 @@ class UserPartition(namedtuple("UserPartition", "id name description groups sche
     # The default scheme to be used when upgrading version 1 partitions.
     VERSION_1_SCHEME = "random"
 
+    RANDOM_SCHEME = "random"
```

Comment:
I think I would move these constants out of partitions.py and put them instead in the only file they are used, /edx-platform/cms/djangoapps/contentstore/views/course.py.

The code in course.py is unfortunately doing a lot of special-case logic for each of these scheme types. I wonder if there is a better way for the code to be structured, as adding yet another scheme type will require a lot of work in course.py. However, for the short term, I would at least suggest isolating knowledge of the scheme types as much as possible to course.py.

The only other place I know where the scheme type is coded is in split_test_module.py, https://github.com/edx/edx-platform/blob/e64d45b7053cdb727f2c305462d16c5ada152406/common/lib/xmodule/xmodule/split_test_module.py#L96 (and the VERSION_1_SCHEME variable above).